src/Validation/Validator.php

```diff
@@ -395,7 +395,7 @@ public function allowEmpty($field, $when = true)
      *
      * ```
      * $message = 'This field cannot be empty';
-     * $validator->notEmpty('email'); // Email cannot be empty
+     * $validator->notEmpty('email'); // Email can be empty
```

Comment:
I must say, I find that naming scheme pretty bonkers.. and counter intuitive.
This would be more sane IMO:
```
$validator->notEmpty('email'); // Email can NOT be empty
$validator->notEmpty('email', true); // Email CAN be empty
```
Or better yet:
```
$validator->allowEmpty('email', false); // Email can NOT be empty
$validator->allowEmpty('email'); // Email CAN be empty
```
Which would effectively do the same, but clearer, since it is now positive (and alings with the positive default as result).

So maybe rename and deprecate notEmpty()?